persistence/src/main/java/com/metrics_board/persistence/entity/Project.java

```diff
@@ -0,0 +1,41 @@
+package com.metrics_board.persistence.entity;
+
+import lombok.*;
+import jakarta.persistence.*;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Entity
+@Table(schema = "roll", name = "project")
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+public class Project {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "project_id_seq")
+    @SequenceGenerator(name = "project_id_seq", sequenceName = "roll.project_id_seq", allocationSize = 1, initialValue = 1000)
+    private Long id;
+
+    @Column(name = "owner_id", nullable = false)
+    private UUID ownerId;
+
+    @Column(name = "name", nullable = false, length = 1024)
+    private String name;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "status", nullable = false, length = 10, columnDefinition = "VARCHAR(10) CHECK (status IN ('active', 'suspended', 'archived'))")
+    private String status;

```

Comment:
Let's define enum `ProjectStatus` instead of String
That also eliminate need  of `columnDefinition` attribute